templates/docs/patterns/hero/index.md

```diff
@@ -33,7 +33,7 @@ This is useful when your hero contents, especially your image, are not suitably
 This makes your hero somewhat safer to use, as it helps to avoid awkward content sizing on medium screens, making all
 content stack vertically.
 
-<div class="embedded-example"><a href="/docs/examples/patterns/hero/hero-50-50" class="js-example">
+<div class="embedded-example"><a href="/docs/examples/patterns/hero/hero-50-50" class="js-example js-show-template">
```

Comment:
I think the `js-` class names should be used mostly as means to identify elements to be targeted via JS, not to effectively pass options to them.

I'd rather see it as `data-` attribute. Something like:
```<div class="embedded-example"><a href="/docs/examples/patterns/hero/hero-50-50" class="js-example" data-lang="jinja">```

Reply:
These have been changed to use `[data-lang='jinja']`